packages/firebase_ml_custom/example/android/app/src/main/res/values/styles.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <!-- Theme applied to the Android Window while the process is starting -->
-    <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
-        <!-- Show a splash screen on the activity. Automatically removed when
-             Flutter draws its first frame -->
-        <item name="android:windowBackground">@drawable/launch_background</item>
-    </style>
-    <!-- Theme applied to the Android Window as soon as the process has started.
-         This theme determines the color of the Android Window while your
-         Flutter UI initializes, as well as behind your Flutter UI while its
-         running.
-         
-         This Theme is only used starting with V2 of Flutter's Android embedding. -->
-    <style name="NormalTheme" parent="@android:style/Theme.Black.NoTitleBar">
-        <item name="android:windowBackground">@android:color/white</item>
-    </style>
+  <!-- Theme applied to the Android Window while the process is starting -->
+  <style name="LaunchTheme" parent="@android:style/Theme.Black.NoTitleBar">
```

Comment:
interesting - are these auto-generated? If so they're not tracking the new DayNight stuff for system dark mode. Not important, just noting it

Reply:
Generated unfortunately